metric-observe-only/metric-observe-only.sh

```diff
@@ -52,10 +52,13 @@ setup_container() {
   # Run as root as the ubuntu (uid 1000) user may not be ready yet.
   Cexec cloud-init status --wait >/dev/null
 
-  # We'll use hyperfine to run the measurement
   Cexec apt-get -q update
+  # We'll use hyperfine to run the measurement
   Cexec apt-get -qy install hyperfine
 
+  # Silence known spikes
+  Cexec apt-get -qy remove unattended-upgrades
```

Comment:
+1 on doing this. Another possibility is `systemctl mask --now unattended-upgrades.service`, which should fully prevent it from running altering the system even less. There's little difference I believe, up to you.

Reply:
I like yours more as uninstalling will be more cpu/disk activity noise.